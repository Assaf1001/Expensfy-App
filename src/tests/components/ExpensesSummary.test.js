import React from "react";
import { shallow } from "enzyme";
import { ExpensesSummary } from "../../components/ExpensesSummary.jsx";

test("Should currenctly render expenses summary with 1 expense", () => {
    const wrapper = shallow(
        <ExpensesSummary expenseCount={1} expenseTotal={235} />
    );
    expect(wrapper).toMatchSnapshot();
});

test("Should currenctly render expenses summary with multiple expenses", () => {
    const wrapper = shallow(
        <ExpensesSummary expenseCount={4} expenseTotal={1000} />
    );
    expect(wrapper).toMatchSnapshot();
});
