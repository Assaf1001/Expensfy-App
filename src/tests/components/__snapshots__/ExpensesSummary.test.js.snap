// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should currenctly render expenses summary with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling
     
    $2.35
  </h1>
</div>
`;

exports[`Should currenctly render expenses summary with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    4
     
    expenses
     totalling
     
    $10.00
  </h1>
</div>
`;
